test/e2e/logforwarding/syslog/forward_to_syslog_test.go

```diff
@@ -0,0 +1,144 @@
+package fluent
+
+import (
+	"fmt"
+	"runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apps "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	logforward "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("LogForwarding", func() {
+	_, filename, _, _ := runtime.Caller(0)
+	logger.Infof("Running %s", filename)
+	var (
+		err              error
+		syslogDeployment *apps.Deployment
+		e2e              = helpers.NewE2ETestFramework()
+	)
+	BeforeEach(func() {
+		if err := e2e.DeployLogGenerator(); err != nil {
+			logger.Errorf("unable to deploy log generator. E: %s", err.Error())
+		}
+	})
+	Describe("when ClusterLogging is configured with 'forwarding' to an external syslog server", func() {
+
+		Context("with the legacy syslog plugin", func() {
```

Comment:
"legacy" may be appropriate if we need to test a new and an old but won't we simply be able to swap out the plugin, update conf, and the essence of the test should remain

Reply:
Yeah, that's what I had in mind - to have an hybrid mode where the legacy one points to the current plugin and "syslog" points to the new plugin, at least until we'll have confidence in the new plugin to provide all the functionality of the current one and at the same quality (performance-wise).
But we can do it as part of the next PR that introduces the new plugin, by then we'll have better feeling of the gaps between the two and whether or not this hybrid mode makes sense - so removed the "legacy" part everywhere